### PR TITLE
Makes it so revs (not revheads) have to wear masks like sling thralls

### PR DESCRIPTION
**Pull request by monster860**
:cl: monster860
tweak: Revs now have to wear masks
/:cl:

### CODE REVIEW

**Comment by ThatLing** [comment]
Kneejerk reaction

**Comment by Bibby0110** [comment]
no, dont touch my revs, also time to metagame the rev heads :thonk:

**Comment by SomeguyManperson** [comment]
i ded

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -254,6 +254,8 @@
 
 	if(!skipface && mind && mind.has_antag_datum(ANTAG_DATUM_THRALL))
 		msg += "[t_His] features seem unnaturally tight and drawn.\n"
+	if(!skipface && is_revolutionary(src) && !is_head_revolutionary(src))
+		msg += "[t_His] face displays a clear, blind hatred for NT.\n"
```
Comment (ThatLing) [inline code/modules/mob/living/carbon/human/examine.dm]: Might want to remove the "for NT" part?

**Comment by Moja10** [comment]
This is real dumb. Like real dumb dumb, consider merging cruix's faces of the revolution though

**Comment by Smudged** [comment]
Faces of revolution by Cruix is good. 

This is seemingly unnecessary, and sort of... bad, the description of "Looking like you hate NT" Is a liiitllee nonsensical.

**Comment by Amelia0010** [comment]
https://github.com/yogstation13/Yogstation-TG/pull/4234 is better

**Pull request closed without merging**